setup.py

```diff
@@ -65,7 +65,7 @@
         "pyhocon==0.3.35",
         "python-Levenshtein==0.12.0",
         "sacremoses",
```

Comment:
This was deleted above.

Reply:
We still need this. There were two lines of this in the file above.